library/swift/src/BUILD

```diff
@@ -21,15 +23,18 @@ swift_static_framework(
         "ResponseTrailersBuilder.swift",
         "RetryPolicy.swift",
         "RetryPolicyMapper.swift",
+        "StatsClient.swift",
+        "StatsClientImpl.swift",
```

Comment:
Same here regarding living in `stats/`

Reply:
same as above, `StatsClient` and its implementation ideally live at the same level as `StreamClient` given the `Engine.swift` interface.